tests/Common/Repositories/AbstractDoctrineRepositoryTest.php

```diff
@@ -31,6 +31,7 @@ protected function setUp(): void
                 __DIR__.'/../../../src/Question/Repositories/Entities',
                 __DIR__.'/../../../src/User/Repositories/Entities',
                 __DIR__.'/../../../src/Company/Repositories/Entities',
+                __DIR__.'/../../../src/Registration/Repositories/Entities',
```

Comment:
@stejes in theorie kunnen we hier de aparte paden vervangen door `__DIR__.'/../../../src/'` maar dan is het uitvoeren van de unit testen trager. Is het bij jou dan ook trager met `$ ./vendor/bin/phpunit`?

Reply:
met `$./vendor/bin/phpunit` is het even traag